packages/esm-patient-attachments-app/src/attachments/camera-upload.component.tsx

```diff
@@ -75,7 +75,13 @@ const CameraUpload: React.FC<CameraUploadProps> = ({ onSavePhoto, onTakePhoto, c
               <label htmlFor="uploadPhoto" className={styles.choosePhoto}>
                 {t('selectPhoto', 'Select local photo instead')}
               </label>
-              <input type="file" id="uploadPhoto" accept="image/*" className={styles.uploadFile} onChange={upload} />
+              <input
+                type="file"
+                id="uploadPhoto"
+                accept="image/*, application/pdf"
+                className={styles.uploadFile}
+                onChange={upload}
+              />
```

Comment:
@jwamalwa, condense the code into a single line at line 78

Reply:
@99sumedha We use prettier to apply automatic formatting to code. While this could be condensed to one line, there's really no advantage in doing do and it means breaking our formatting conventions.

Reply:
Thanks so much for review am going to fix it shortly.